crates/tools/Cargo.toml

```diff
@@ -15,10 +15,10 @@ anyhow = "1.0"
 thiserror = "1.0"
 toml = "0.5"
 tempfile = "3.1"
-lazy_static = "1.3"
+lazy_static = "1.4"
 secp256k1 = "0.24"
 sha3 = "0.10.6"
-reqwest = { version = "0.11", features = ["json"] }
+reqwest = { version = "0.11", features = ["json", "blocking"] }
```

Comment:
Is blocking client actually used?

Reply:
Yes, this feature is enabled in gw-dynamic-config crate, since I removed the crate in this pr, I need to add the feature back to make compiling work.